cmake/SetBuildType.cmake

```diff
@@ -1,7 +1,52 @@
 # Distributed under the MIT License.
 # See LICENSE.txt for details.
 
-set(CMAKE_BUILD_TYPES "Debug" "Release" "RelWithDebInfo" "MinSizeRel")
+# Build type "None" represents what CMake does by default if you do not specify
```

Comment:
I'm confused by this first line. If I run `cmake /path/to/spectre` what build type will I get, `NONE` or `DEBUG`?

Reply:
Sorry, I meant CMake out of the box, not with our modifications below.  I will clarify